src/main/deploy/shooterData.csv

```diff
@@ -0,0 +1 @@
+Distance,Angle,Power
```

Comment:
Consider adding a couple lines of placeholder values to verify the CSV parser works as expected.